### PR TITLE
Milestone 1: Plain JavaScript with objects

### PR DESCRIPTION
**Pull request by grauJavier**
# 🌟 Features
- **Book List:** Display a list of books with title and author.
- **Add Book:** Enable users to enter book details via an input form and dynamically add new entries to the list.
- **Remove Book:** Allow users to remove books from the list using a delete button or icon.
- **Dynamic DOM Manipulation:** Update the webpage instantly without page reloads using JavaScript to manipulate the DOM.
- **JavaScript Objects and Arrays:** Store and manage book data using objects and arrays for easy retrieval, addition, and removal of entries.

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @grauJavier and @Dachrono,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:


- [ ] Some linters checks are not passed. Please make sure that all checks are passing and green.
- [ ] This is a peer-programming project milestone and I can see only two team member has committed all changes, So please make sure that all team members have the same amount of work shared and committed to GitHub. If your partner isn't responding, Please contact student success about it and get approval that only both of you work on this project without him and add a screenshot of this reply on the Pull Request summary.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by prolajumokeoni** [comment]
Hi Team,
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a collaboration  project, that does not have commit from all partners, please check previous reviewer invalidation message 👍🏿  
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Dachrono** [comment]
Hi, good day 😄, @prolajumokeoni Smas has never answered, by any means, we send the situation to student success but we have not received any response either 😢

**Comment by prolajumokeoni** [comment]
> Hi, good day 😄, @prolajumokeoni Smas has never answered, by any means, we send the situation to student success but we have not received any response either 😢

Please when you get a response from Student success add a screenshot to this pr

**Comment by grauJavier** [comment]
image

File: JS/script.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+// 4.0 SAVE DATA
+//   4.1 Function to save data
+function saveData() {
+  if (storageAvailable('localStorage')) {
+    const newBook = {
+      title: titleInp.value,
+      author: authorInp.value,
+    };
+
+    bookList.push(newBook);
```
Comment (fobadara) [inline JS/script.js:81]: [OPTIONAL] It is better to use an array method that does not mutate the array like Concat in this case. This helps the memory and speed of your app

**Review by fobadara — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Clean code
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**